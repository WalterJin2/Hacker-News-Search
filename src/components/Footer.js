

const Footer = () => {

     const content = (
       <div className="flex flex-col justify-between items-center  w-full px-1 py-1">
         <div className="px-1 py-1">Copyright Â© 2022 Walter Jin</div>
         <div className="px-1 py-1">V1.0</div>
       </div>
     );
    return content
}
export default Footer