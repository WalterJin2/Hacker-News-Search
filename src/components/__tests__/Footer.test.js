import Footer from "../Footer";
import { renderWithProviders } from "../../test/test-utils";

it("Footer renders correctly", () => {
  const { asFragment } = renderWithProviders(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});