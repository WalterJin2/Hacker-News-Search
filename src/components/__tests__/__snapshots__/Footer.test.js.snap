// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col justify-between items-center  w-full px-1 py-1"
  >
    <div
      class="px-1 py-1"
    >
      Copyright Â© 2022 Walter Jin
    </div>
    <div
      class="px-1 py-1"
    >
      V1.0
    </div>
  </div>
</DocumentFragment>
`;
